sdk/storage/Azure.Storage.Files.Shares/tests/FileClientTests.cs

```diff
@@ -1632,6 +1632,33 @@ public async Task StartCopyAsync_CopySourceFileLastWrittenOnError()
                 e.Message));
         }
 
+        [RecordedTest]
+        public async Task StartCopyAsync_CopySourceFileChanagedOnError()
```

Comment:
NIT Typo: change StartCopyAsync_CopySourceFileChanagedOnError to
StartCopyAsync_CopySourceFileChangedOnError